### ISSUE TITLE
Unable to install OS from USB on Razer Blade 15 (2022)

### ISSUE THREAD

**Issue by johnnymatthews** [pop-os/beta#398]
I was unable to install this beta on my 2022 Razer Blade 15 with the following specs:

CPU: i7-12800H
RAM: 16 GB
GPU: NVidia GeForce RTX 3070 Ti (8GB)

This laptop has an option to enable _dynamic_ GPU mode; essentially it turns off the GPU and uses the CPU's internal graphics when there isn't a significant GPU load. I'm guessing this is in an effort to keep temps down and reduce power draw. In the UEFI there is an option to disable this feature, and keep the discrete (GPU) graphics enabled at all times. In the table below I've listed this as either _dynamic_ or _discrete_.

I ran the install a few different times with different UEFI/BIOS configurations. Secure Boot was disabled for all installation attempts. In each attempt, if the installer/loader hung, I waited for 5 minutes to see if anything else would happen. Here's what I found:

| Pop_OS ISO version | GPU mode | Fast Boot | Status | Notes |
| --- | --- | --- | --- | --- |
| Intel/AMD | Dynamic | Enabled | Failed | Reached the following line: `nouveau [...] timer stalled at ffffffffffff`. |
| Intel/AMD | Dynamic | Disabled | Failed | Same as previous result. |
| Intel/AMD | Discrete | Enabled | Reached installer | The installer defaulted to HDPI mode with the scaling at 200%. If I tried to open **Settings** or right click the desktop and open **Display settings**, I was logged out of the Live User Account and sent to the login screen. Following the installer all the way through works, but on first boot of the OS, I am repeatedly kicked back out to the login screen. The login screen is scaled at 100%. |
| Intel/AMD | Discrete | Disabled| Reached installer | The same results as the previous attempt. |

| Pop_OS ISO version | GPU mode | Fast Boot | Status | Notes |
| --- | --- | --- | --- | --- |
| Nvidia | Dynamic | Enabled | Failed | Loader got stuck on `Started RealtimeKit scheduling Policy Service` |
| Nvidia | Dynamic | Enabled | Disabled | Same as previous result. |
| Nvidia | Discrete | Enabled | Reached installer-ish| Reaches the installer, but no GUI is shown. The background is grey, and the mouse is visible. I can move the mouse around the screen. I cannot reach the terminal from here. |
| Nvidia | Discrete | Disabled| Reached installer-ish| The same results as the previous attempt. |

**Other notes**

I'm happy to do more digging, but I don't know the steps on how to debug this kind of problem. Any advice is welcome!

**Comment by xploz1on** [comment]
It would be hard to find a solution to your problem here. I believe you would have better luck in [Razer Linux forum](https://insider.razer.com/index.php?forums/linux/)

I am guessing that you dont have any problem whatsoever running Windows correct?
Have you tried another Linux distro?

A quick search around tells me its kinda tricky to setup a daily driver linux on some Razer laptops.

**Comment by johnnymatthews** [comment]
> dont have any problem whatsoever running Windows correct?

Correct. Windows 11 runs fine. Windows 10, however, does not run since Razer hasn't released any signed drivers for that OS. 

> Have you tried another Linux distro?

I've tried Ubuntu 20.04 which mostly worked fine. I also tried Manjaro (latest build) that ran straight out of the box.

> some Razer laptops.

Yeah... I had a Razer Blade Stealth (2019) and that Pop_OS like a dream! It was excellent! Ah well. I'm hoping that it's likely a driver issue with the 3080 TI in this laptop. Hopefully there might be a driver update in the near future that gets me to the install screen.
